operations/helm/charts/mimir-distributed/templates/graphite-proxy/graphite-write-proxy/graphite-write-proxy-dep.yaml

```diff
@@ -53,6 +53,7 @@ spec:
             - -target=graphite-write-proxy
             - -config.file=/etc/mimir/mimir.yaml
             - -license.path=/license/license.jwt
+            - -instrumentation.enabled=false
```

Comment:
Out of curiosity - why is this needed?

Reply:
To avoid a common error in metrics enterprise image i.e
```level=warn ts=2022-08-24T06:54:40.097647Z caller=gatherer.go:71 msg="Unable to send system metrics" err="unable to push to distributor: rpc error: code = Unavailable desc = last resolver error: produced zero addresses"````

The instrumentation is always enabled as it thinks its always going to run some mimir component and expose cortex metrics but it is not the case when running graphite targets.

Reply:
Just reenabled instrumentation